### PR TITLE
Fixes even more controller tests to use keyword arguments.

### PR DESCRIPTION
**Pull request by ashercodeorg**
All of HomeControllerTest, LevelSourcesControllerTest, LevelsControllerTest, MediaProxyControllerTest, NotesControllerTest.

### CODE REVIEW

File: dashboard/test/controllers/level_sources_controller_test.rb [hunk 1]
```diff
@@ -196,41 +196,4 @@ def expect_s3_upload
       apple_mobile_web_app: true
     )
   end
-
-  test 'migrates old flappy levels' do
```
Comment (breville) [inline dashboard/test/controllers/level_sources_controller_test.rb:200]: Why are these being deleted?

**Comment by breville** [comment]
one q, otherwise lgtm.

File: dashboard/test/controllers/level_sources_controller_test.rb (hunk 1 above)
Comment (ashercodeorg) [inline dashboard/test/controllers/level_sources_controller_test.rb:200]: This test is being deleted so as to not cause a merge conflict with another pending PR that deletes it. Really, I should deal with the merge, but given how many PRs I'm cranking out to fix these warnings, I'm too lazy.

Comment (ashercodeorg) [reply dashboard/test/controllers/level_sources_controller_test.rb:200]: Per the (newly added) comment, to prevent merge conflicts.

**Pull request merged**